src/components/ManaCalculator/hooks/useResults.ts

```diff
@@ -42,9 +42,14 @@ export function useResults(state: ManaCalculatorProps) {
   const grantedTPS = calculateTPS(manaGenerated, state.congestion);
   const totalTPS = grantedTPS + additionalTPS;
 
+  const timeToGetEnoughMana = 1 / totalTPS;
+  const timeToGetEnoughManaAsHolder = 1 / additionalTPS;
```

Comment:
I think it would be beneficial specifying somehow that these are in seconds

Reply:
Changed in beca8fd658bcf7fae747dbfd0414377fd84534c9